### ISSUE TITLE
In larger tables, columns become collapsed on top of each other

### ISSUE THREAD

**Issue by stevenlockton-progenity** [line/conflr#137]
Hello,

When tables (printed by `knitr::kable()`) become a certain size some columns collapse together. It's an odd bug that has only recently become apparent.

Here's what I expect (3 x 10 table example used)
![image](https://user-images.githubusercontent.com/43186084/134224914-46990269-93f4-4462-b908-18448953d1d1.png)

And here's what happens (22 x 50 table example)
![image](https://user-images.githubusercontent.com/43186084/134225094-e75aa923-4893-4c57-b00d-dc8e46adfaea.png)

Here is some code to reproduce the bug that can be used in a `conflr` `confluence_document`.

```{r}
make_fake_data_table <- function(nrows, ncolumns) {
  set.seed(123)
  n <- ncolumns * nrows
  mx <- matrix(
    data = rnorm(n),
    nrow = nrows,
    ncol = ncolumns
  )
  df <- data.frame(
    A = 1:nrows,
    B = letters[1:nrows],
    C = factor(LETTERS[1:nrows]),
    mx
  )
  return(df)
}

fake_dt <- make_fake_data_table(
  nrows = 22,
  ncolumns = 50
)

knitr::kable(fake_dt)
```

At some point this issue goes away - it's not a simple "below x columns it's fine" because it also depends on the row number too.

Any ideas on what's causing this?

**Comment by yutannihilation** [comment]
Thanks for reporting! I have no idea on conflr's side. If you manually create 22 x 50 table on editor, what will happen? As you say

> It's an odd bug that has only recently become apparent.

I'm wondering if this is something on Confluence's side.

**Comment by stevenlockton-progenity** [comment]
Hey - I copy and pasted a big table full of random numbers from excel with no issues. I haven’t manually created a table with confluences table tool. I’ll try.

I’m guessing it’s something to do with atlassian’s recent api updates. Is there anything in the md file that you can see that conflr would struggle with?

S


________________________________

**Comment by yutannihilation** [comment]
> I’m guessing it’s something to do with atlassian’s recent api updates.

Ah, this sounds possible... Thanks. I know one recent API change on Confluence Cloud https://github.com/line/conflr/issues/125#issuecomment-921688057, they changed the default editor to the new one, so it might matter here. Do you use Confluence Cloud?

**Comment by stevenlockton-progenity** [comment]
Yes we do. We preferred the old editor, it’s more compact and allows for code hiding. Is there a way to optionally use the old editor in conflr now the new editor is the default?

S
________________________________

**Comment by yutannihilation** [comment]
No option at the moment. I need to implement the feature to switch between new and old, but I couldn't find time for this, sorry. Still not sure if using the old editor will solve this issue, but I'll try to implement it, hopefully soon. Feel free to ping me for this!

**Comment by stevenlockton-progenity** [comment]
I’d be very interested in that feature. Atlassian are painfully slow to fix anything so I’d rather switch back to what works than rely on them to fix the new default.
________________________________

**Comment by stevenlockton-progenity** [comment]
If you need help with this feature (namely, selecting which editor to use when publishing to confluence cloud), let me know. I might be able to assist. When do you think you'd be able to work on this?

**Comment by yutannihilation** [comment]
No idea, hopefully next week, or next month :)
